source/graphql/types-and-resolvers.txt

```diff
@@ -155,6 +155,18 @@ GraphQL types that they map to:
    
    * - ``timestamp``
      - ``DateTime``
+   
+   * - ``undefined``
+     - ``{}``
+
+.. note::
+   
+   If a document field is explicitly set to ``undefined`` then the
```

Comment:
It might help to have a brief contextualization such as "JS has two types representing 'no value' (undefined and null) while GraphQL only supports one (null)" or whatever is the reason for this.